### ISSUE TITLE
nvda does not announce text navigation after pressing a menu option in balabolka 

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#13046]
### Steps to reproduce:
open the balabolka app
type a text
press alt, navigate with right arrows to the menu
speech
press down arrow to be in menu option
read in audible voice
press enter.
nvda informs the panel wrongly
now try to navigate the text by pressing arrows, and also try to delete the typed text by pressing backspace or delete
Text is deleted indicating that focus is on the text field. 
### Actual behavior:
nvda does not report hover characters and neither does it report deleted characters.
After clicking on the menu option, nvda also informs the panel.
The focus is on the text field, because although nvda doesn't say the focus is again on the text field and the text is edited 
### Expected behavior:
when hitting enter in the menu option nvda should inform editing.
nvda must inform the deleted characters, or the character in focus when navigating with the arrows
It seems that for some reason nvda does not correctly recognize the focus in the edit field after the focus leaves the menu option.
we can emphasize that when removing the focus from the editing window and returning the focus to the editing window, the text is read normally. 
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
NVDA version alpha-24184,fd9ae8c4
#### Windows version:
11 22000.282
#### Name and version of other software in use when reproducing the issue:

Balabolka version 2.15.0.804
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
log:
[blb.txt](https://github.com/nvaccess/nvda/files/7508328/blb.txt)
